modules/new_profile/php/new_profile.class.inc

```diff
@@ -44,19 +53,278 @@ class New_Profile extends \NDB_Page
         return false;
     }
     /**
-     * Include additional JS files:
-     *  1. editForm.js - reactified form to update media
+     * Processes the values and saves to database
+     *
+     * @param array $values form values
+     *
+     * @return void
+     */
+    function _process($values): void
+    {
+        // Change empty strings to null before passing as arguments.
+        foreach ($values as $key => $value) {
+            if (empty($value)) {
+                $values[$key] = null;
+            }
+        }
+        /* When a user has only one site, or when a centerID has not been
+         * passed, choose the first one in the User's centerID list.
```

Comment:
I think this should only be true if the user has 1 site. If it has more, it should be an error to not pass one.